build-system/tasks/compile-jison.js

```diff
@@ -43,16 +43,16 @@ const imports = new Map([
  * @return {!Promise}
  */
 async function compileJison(searchDir = jisonPath) {
-  fs.mkdirSync('build/parsers', {recursive: true});
-
-  const promises = globby.sync(searchDir).map((jisonFile) => {
-    const jsFile = path.basename(jisonFile, '.jison');
-    const extension = jsFile.replace('-expr-impl', '');
-    const parser = extension + 'Parser';
-    const newFilePath = `build/parsers/${jsFile}.js`;
-    return compileExpr(jisonFile, parser, newFilePath);
-  });
-  return Promise.all(promises);
+  const jisonFiles = globby.sync(searchDir);
```

Comment:
While we're at it:
```suggestion
  const jisonFiles = await globby(searchDir);
```

Reply:
Good catch. There are several more of these in `build-system/` that can be made asynchronous.

Reply:
Done in #35723.